### PR TITLE
add DolphinScheduler slack channel

### PR DESCRIPTION
**Pull request by dailidong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ It is because of the shoulders of these open source projects that the birth of t
 ### Get Help
 1. Submit an issue
 1. Subscribe the mail list : https://dolphinscheduler.apache.org/en-us/docs/development/subscribe.html.  then send mail to dev@dolphinscheduler.apache.org
+1. Slack channel: [Slack channel](https://join.slack.com/share/zt-do3gvfhj-UUhrAX2GxkVX_~JJt1jpKA)
```
Comment (zhongjiajie) [inline README.md:102]:
```suggestion
1. Slack channel: [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://join.slack.com/share/zt-do3gvfhj-UUhrAX2GxkVX_~JJt1jpKA)
```

Comment (zhongjiajie) [reply README.md:102]: How about using badge instead of link?

**Comment by zhongjiajie** [comment]
Screen Shot 2020-04-25 at 22 56 42

Just like this.

File: README.md (hunk 1 above)
Comment (dailidong) [reply README.md:102]: good idea，haha

**Comment by davidzollo** [comment]
> Screen Shot 2020-04-25 at 22 56 42
> 
> Just like this.

good idea, I have merge your update.  I think we can do some deep communication about workflow,  can you add my mail(lidongdai@apache.org) or add wechat(510570367)， when added ,please tell me your github id so that I could recognize you , thx

**Comment by zhongjiajie** [comment]
@dailidong Add it

**Comment by kezhenxu94** [comment]
image

@dailidong @qiaozhanwei

**Pull request merged**